<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileListWidget</class>
 <widget class="QWidget" name="FileListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="fileListView"/>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Remove file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
